sonar-dependency-check-plugin/src/main/java/org/sonar/dependencycheck/DependencyCheckConfiguration.java

```diff
@@ -48,37 +48,32 @@ public static List<PropertyDefinition> getPropertyDefinitions() {
                         .description("path to the 'dependency-check-report.html' file")
                         .defaultValue(DependencyCheckConstants.HTML_REPORT_PATH_DEFAULT)
                         .build(),
-                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_BLOCKER)
+                PropertyDefinition.builder(DependencyCheckConstants.SEVERITY_HIGH)
```

Comment:
Please set the configuration option `deprecatedKey` to the old Critical configuration key.